ipfshttpclient/version.py

```diff
@@ -8,4 +8,5 @@
 # `0.4.1` and so on. When IPFS `0.5.0` is released, the first client version
 # to support it will also be released as `0.5.0`.
 
-__version__ = "0.7.0"
+__version__ = "0.8.0a1"
```

Comment:
Why 0.8.0 though?

Reply:
I saw you had done that with a `0.7` previously so I followed suit. Also erring on the side of caution, since this would be my first time running the publish process.

Shouldn't be hard to republish `0.8.0` after publishing `0.8.0a1` if all is well with `0.8.0a1`.

Reply:
Fair enough :slightly_smiling_face: